components/utils/getAnglePercentageObject.js

```diff
@@ -0,0 +1,58 @@
+import { getPercentageFromAngle } from './index'
```

Comment:
Follow a consitent coding pattern. This example has used semicolons to end sentences. You must follow it.

Reply:
semicolons added.